kolibri/plugins/style_guide/assets/src/views/content/api_table.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+
+  <div>
+
+    <h3>Name: {{ api.name }}</h3>
+    <h3>Description: {{ api.description }}</h3>
+
+    <div v-if="api.props.length">
+      <h3>Props</h3>
+      <table>
+      <tr>
+        <th>Name</th>
+        <th>Type</th> 
+        <th>Required</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr v-for="prop in api.props">
+        <td>{{ prop.name }}</td>
+        <td>{{ prop.value.type }}</td>
+        <td>{{ prop.value.required }}</td>
+        <td>{{ prop.value.default ? prop.value.default : 'None' }}</td>
```

Comment:
maybe `undefined` or just a `–` instead of `None`? (it sounds like the Python keyword)